zerver/lib/mobile_auth_otp.py

```diff
@@ -25,11 +24,11 @@ def ascii_to_hex(input_string: str) -> str:
 
 def hex_to_ascii(input_string: str) -> str:
     """Given a hex array, decode it back to a string"""
-    return force_str(binascii.unhexlify(input_string))
+    return binascii.unhexlify(input_string).decode('utf8')
```

Comment:
As per the documentation,( https://docs.python.org/2/library/binascii.html),  binascii.unhexlify return binary data.